src/core/render/index.js

```diff
@@ -10,6 +10,7 @@ import { scrollActiveSidebar } from '../event/scroll';
 import { Compiler } from './compiler';
 import * as tpl from './tpl';
 import { prerenderEmbed } from './embed';
+import DOMPurify from 'dompurify';
```

Comment:
Hmmm... Just an idea, but, what if we use the builtin `DOMParser` to load the HTML, then we can use `querySelector` to find bad parts and remove them, then finally stick the resulting `Node` into the DOM.

This would eliminate the need for the library.

Reply:
Taking a look at this now.